PROJECT

```diff
@@ -1,42 +1,51 @@
 domain: openshift.io
-layout: go.kubebuilder.io/v2
+layout:
+- go.kubebuilder.io/v3
+plugins:
+  manifests.sdk.operatorframework.io/v2: {}
+  scorecard.sdk.operatorframework.io/v2: {}
 projectName: ocs-operator
-repo: github.com/red-hat-storage/ocs-operator
+repo: github.com/red-hat-storage/ocs-operator/v4
```

Comment:
I think the repo link won't contain the /v4, going to that url gives a 404 error.

Reply:
We sorted this over a call